invenio_users_resources/config.py

```diff
@@ -240,3 +240,6 @@ class OrgPropsSchema(Schema):
 
 USERS_RESOURCES_DOMAINS_ORG_SCHEMA = OrgPropsSchema
 """Domains organisation schema config."""
+
+USERS_RESOURCES_GROUPS_ENABLED = True
```

Comment:
This needs to be very well documented and our colleagues from Zenodo need to put it to `False`, let's don't forget it :)